<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar cart√£o</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cartao</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNumeroCartao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>37</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#txtNumeroCartao{
border:none;
border-bottom:1px solid #f27e13;
padding:5px;
}</string>
    </property>
    <property name="placeholderText">
     <string>Ex. 9065</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGravar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnGravar{
border:1px solid #f27e13;
padding:5px;
background-color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>Gravar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigoCartao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#txtCodigoCartao{
border:none;
border-bottom:1px solid #f27e13;
padding:5px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnCancelar{
border:1px solid #f27e13;
padding:5px;
background-color:#f27e13;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
